contracts/zero-ex/contracts/src/features/TransformERC20Feature.sol

```diff
@@ -244,6 +281,12 @@ contract TransformERC20Feature is
                     args.recipient
                 );
             }
+            // Transfer output tokens from wallet to recipient
+            outputTokenAmount = _executeOutputTokenTransfer(
```

Comment:
So this would replace `PayTakerTransformer`? That transformer transfers more than the output token currently, like leftover taker token (buys) or unspent protocol fees so we would still need it. Maybe we could pass in additional tokens to transfer out.

Also does the fact that we set `minOutputTokenAmount = 0` [in multiplex](https://github.com/0xProject/protocol/blob/feat/otc/spot-check/contracts/zero-ex/contracts/src/features/multiplex/MultiplexTransformERC20.sol#L47) mean none of this matters in the context of multiplex?

Reply:
A big part of me really wants to just add `outputToken, minBuyAmount` to the transformer context and do the check in (the last) FQT and not be too worried about the return value of `transformERC20()` being gamable because it wouldn't be worth tricking it anymore with the check in FQT.

Reply:
it wouldn't necessarily replace `PayTakerTransformer` (though I had the same thought). It would mean that `PayTakerTransformer` would no longer handle the `outputToken` in particular. It matters to Multiplex insofar as Multiplex can now trust the `outputTokenAmount` returned by `_tranformERC20`

Reply:
I'm not really seeing the benefit of doing that in FQT. I started doing that, but calculating the desired `minBuyAmount` in `MultiplexTransformERC20` is kinda annoying

Reply:
Yeah I guess we would always want `minBuyAmount: 0` for a transformERC20 in a batch fills which would disable any check in the FQT. :thinking: